src/PostalRegistry.Api.Legacy/PostalInformation/Responses/PostalInformationShaclShapeResponse.cs

```diff
@@ -0,0 +1,97 @@
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Threading.Tasks;
+
+namespace PostalRegistry.Api.Legacy.PostalInformation.Responses
+{
+
+    [DataContract(Name = "PostalInformationShaclShape", Namespace = "")]
+    public class PostalInformationShaclShapeReponse
+    {
+        //TODO provide link to collection
+
+        [DataMember(Name = "@id", Order = 2)]
+        [JsonProperty(Required = Required.Always)]
+        public Uri Id { get; set; }
+
+        [DataMember(Name = "@type", Order = 3)]
+        [JsonProperty(Required = Required.Always)]
+        public readonly string Type = "sh:NodeShape";
+
+        [DataMember(Name = "@context", Order = 1)]
```

Comment:
Wouldn't it be nicer to also put this property on top if it is Order 1, just for clarity. This is the stage you can do it without worry